_rules/meta-viewport-b4f0c3.md

```diff
@@ -24,20 +24,34 @@ acknowledgments:
 
 This rule applies to each `content` attribute on a `meta` element with a `name` [attribute value][] of `viewport` for which at least one of the following is true:
 
-- the `content` [attribute value][] has the `user-scalable` property; or
-- the `content` [attribute value][] has the `maximum-scale` property.
+- the `content` attribute specified value includes the text `user-scalable`; or
+- the `content` attribute specified value includes the text `maximum-scale`.
+
+**Note:** The Applicability refers to string inclusion of the indicated strings in the value of the `content` attribute, as written.
 
 ## Expectation 1
 
-For each test target, the [attribute value][] does not have a `user-scalable` property with a value of `no`.
+For each test target, the [attribute value][] does not have a `user-scalable` property with a value of `fixed`.
```

Comment:
Should this have been `user-zoom`?

I'm a little hesitant for us to start using the CSS property names / values here in place of the actual key/value pairs on the content attribute. It creates adds a layer of abstraction to the rule 
which makes it more difficult to understand. Since rules on how to process these props are pretty easy, I would suggest we put them straight into the rule in the same way that the autocomplete rule has a pseudo-copy of the HTML spec's description of what makes a valid autocomplete attribute value.

```suggestion
For each test target, when the [attribute value][] is parsed into key/value pairs, one of the following is true:
- there is no `user-scalable` key; or
- the `user-scalable` value is `yes`; or
- the `user-scalable` value is a number less than -1, or greater than 1.
```

We can then put something in the background about how the expectation follows the steps described in CSS. You also wouldn't have to add those translate statements to the example descriptions.

Reply:
Good idea. I gave it a shot 👍